src/core/space_interaction_API.jl

```diff
@@ -14,6 +14,7 @@ fields should be supplied. See the top of src/core/agents.jl for examples.
 export move_agent!,
     add_agent!,
     add_agent_pos!,
+    add_agent_own_pos!,
```

Comment:
If we have the `own` version, the `pos` version without `own` should be moved to the deprecation file.

Reply:
(I am referring to the export statement above the new name)